src/myr/Myr.js

```diff
@@ -514,10 +534,11 @@ class Myr {
         let base = {
             geometry: "primitive: box;",
             id: "box" + this.genNewId(),
-            material: `color: ${this.cursor.color};`,
+            material: `color: ${this.cursor.color}; side: double; src: ${this.cursor.texture};`,
```

Comment:
These should be consistent across all geometries as to what order the material properties are being set.